website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Judging Open Source by Github Stars
+tags: [open-source]
+authors: yassin
```

Comment:
not sure about how this works, what should I type here? and how it gets map to my fullname and profile picture? 🤔

Reply:
There's an `authors` file where you can add your info

Reply:
And then, the key should map to what you type here. Ie you should add `yassin` to the `authors` object